rpc_jobs/build_summary.yml

```diff
@@ -57,17 +19,37 @@
           days-to-keep: 3
     dsl: |
       library "rpc-gating@${RPC_GATING_BRANCH}"
+      common.shared_slave{
+        env.RE_JOB_REPO_NAME = "rpc-gating"
+        dir("rpc-gating/scripts/build_summary"){
+          String sha = sh(script: "git rev-parse --verify HEAD", returnStdout: true).trim()
+          image = docker_utils.pullOrBuild(IMAGE_NAME, sha)
```

Comment:
This can't be run on a shared slave, as it configures the VPN, which is system wide, and may conflict with other jobs.

So either this can be run on a single use slave of some type, or we can add a Kronos VPN service to the master as we currently have for Phobos, then the whole job can run on the master.